app/src/main/java/com/h/pixeldroid/fragments/dummy/DummyContent.kt

```diff
@@ -0,0 +1,57 @@
+package com.h.pixeldroid.fragments.dummy
+
+import java.util.ArrayList
+import java.util.HashMap
+
+/**
+ * Helper class for providing sample content for user interfaces created by
+ * Android template wizards.
+ *
+ * TODO: Replace all uses of this class before publishing your app.
+ */
+object DummyContent {
```

Comment:
supprime ce fichier. Faut pas merge aveuglement...

Reply:
C'est pas toi qui a mis ce truc sur master à la base ? The comment goes both ways

Reply:
C'est pas sur master, c'est qqch qui reste de la branche de Marie d'un ancien merge (resté du coup en merge avec une nouvelle version qui l'avait plus)